### PR TITLE
Feature/312 bulk activate

### LINKED ISSUE #312: Activate/Deactivate workflow - bulk
Activate/Deactivate workflow - bulk

User should be able to select multiple workflows in datagrid and apply activate/deactivate on all of them
+ probably expose new endpoint for this.

### PR DESCRIPTION
**Pull request by kevinwallimann**
Closes #312

### CODE REVIEW

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html [hunk 1]
```diff
@@ -66,6 +66,19 @@ <h3 class="modal-title">Upload workflow</h3>
     </div>
   </clr-dg-action-bar>
 
+  <clr-dg-action-bar>
```
Comment (jozefbakus) [inline ui/src/app/components/workflows/workflows-home/workflows-home.component.html]: I dont like appearance. I would put it on one line with other buttons and moved all bulk actions under one dropdown.
```      
<clr-dropdown>
        <button type="button" class="btn btn-sm btn-secondary" [disabled]="selected.length <= 0" clrDropdownTrigger>
          Bulk or something better
          <clr-icon shape="caret down"></clr-icon>
        </button>
        <clr-dropdown-menu clrPosition="bottom-left" *clrIfOpen>
          <button type="button" clrDropdownItem>Export All</button>
          <button type="button" clrDropdownItem>Export Selected Items</button>
        </clr-dropdown-menu>
      </clr-dropdown>
```

+ logic of active/deactive I think should be like this: if there is at least one selected with not active then "activate" button should be enabled 
and when there is at least one selected active then "deactive" button should be enabled

Comment (kevinwallimann) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html]: Yes and yes, you're right

Comment (jozefbakus) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html]: 👍🏻

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html [hunk 2]
```diff
@@ -43,9 +43,26 @@ <h3 class="modal-title">Upload workflow</h3>
   </div>
 </clr-modal>
 
-<clr-datagrid (clrDgRefresh)="onClarityDgRefresh($event)">
+<clr-datagrid [(clrDgSelected)]="selected" (clrDgRefresh)="onClarityDgRefresh($event)">
 
   <clr-dg-action-bar>
+    <clr-dropdown>
```
Comment (jozefbakus) [inline ui/src/app/components/workflows/workflows-home/workflows-home.component.html:49]: You can click on disabled button.

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html [hunk 3]
```diff
@@ -43,9 +43,26 @@ <h3 class="modal-title">Upload workflow</h3>
...
+        Bulk Actions
+        <clr-icon shape="caret down"></clr-icon>
+      </button>
+      <clr-dropdown-menu clrPosition="bottom-left" *clrIfOpen>
+        <div aria-label="Activate" clrDropdownItem [disabled]="isActivateSelectedWorkflowsDisabled(selected)" (click)="activateSelectedWorkflows(selected)">
+          <clr-icon shape="power"></clr-icon>
+          Activate
+        </div>
+        <div aria-label="Deactivate" clrDropdownItem [disabled]="isDeactivateSelectedWorkflowsDisabled(selected)" (click)="deactivateSelectedWorkflows(selected)">
+          <clr-icon shape="power"></clr-icon>
```
Comment (jozefbakus) [inline ui/src/app/components/workflows/workflows-home/workflows-home.component.html]: I am sorry that I am pointing this out here ... Icons are the same `shape="power"`. I would distinguish it with this class: `<clr-icon shape="power" class="has-badge"></clr-icon>`. 
Screenshot 2020-09-29 at 12 44 36

+ update single activate/deactivate actions with badge

File: src/main/scala/za/co/absa/hyperdrive/trigger/Application.scala [hunk 4]
```diff
@@ -85,6 +85,7 @@ object ObjectMapperSingleton {
     .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
     .configure(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES, false)
     .configure(DeserializationFeature.FAIL_ON_NULL_CREATOR_PROPERTIES, true)
+    .configure(DeserializationFeature.USE_LONG_FOR_INTS, true)
```
Comment (jozefbakus) [inline src/main/scala/za/co/absa/hyperdrive/trigger/Application.scala]: I think we've already discussed this "bug" and I went for `@JsonDeserialize` annotation see our whatsapp and see `JobsToRun` class.

I am not sure, right now its fine and I tried all endpoints and it's working fine

Comment (kevinwallimann) [reply src/main/scala/za/co/absa/hyperdrive/trigger/Application.scala]: Ah yes I didn't remember. Thanks for pointing it out. Will change it to be consistent

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html (hunk 2 above)
Comment (kevinwallimann) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html:49]: 🤦

Comment (jozefbakus) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html:49]: Yeah :D

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html (hunk 3 above)
Comment (jozefbakus) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html]: I think you missed this one : update single activate/deactivate actions with badge

**Review by jozefbakus — APPROVED** [review]

**Pull request merged**